script.user.js

```diff
@@ -1,10 +1,10 @@
 // ==UserScript==
 // @name         Flyff Universe Multi-Tab Spell Caster
-// @namespace    pPrecel
-// @version      2.0
+// @namespace    pPrecel, doodlefudge
+// @version      1.0
```

Comment:
Please, can you change it to `2.1`? The first version of this tool ( before my rework ) was called `1.0` and it would be really confusing to go back to this version again.
```suggestion
// @version      2.1
```